src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--main-color);
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: block;
+    margin-top: 32px;
+    width: 160px;
+  }
+
+  &__title {
+    font-size: var(--size);
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+  }
+
+  &__label {
+    font-size: var(--min-size);
+    line-height: var(--min-line-height);
+
+    margin-bottom: 16px;
+    color: var(--secondary-color);
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+
+    margin-top: 16px;
+    margin-bottom: 24px;
```

Comment:
use margin in one direction only to avoid margin collapse.
If the upper block will contain margin-bottom - indents will not work properly(`margin-top` and `margin-bottom` will overlap each other)

```
    < -- block -- >
< -- margin-bottom -- >
    < -- block -- >
< -- margin-bottom -- >

```